docs/02.API-REFERENCE.md

```diff
@@ -7731,22 +7468,22 @@ The behaviour of the Proxy can be specialized with an options argument.
 
 *Note*:
   - This API depends on the es.next profile.
-  - Returned value must be freed with [jerry_release_value](#jerry_release_value)
+  - Returned value must be freed with [jerry_value_free](#jerry_value_free)
     when it is no longer needed.
 
 **Prototype**
 
 ```c
 jerry_value_t
-jerry_create_special_proxy (const jerry_value_t target,
-                            const jerry_value_t handler,
-                            uint32_t options);
+jerry_proxy_special (const jerry_value_t target,
```

Comment:
`jerry_proxy_custom` and `jerry_proxy_object_options_t`-> `jerry_proxy_custom_behavior_t`